packages/frontend/src/router/index.js

```diff
@@ -141,6 +141,45 @@ const routes = [
           }
         ]
       },
+      {
+        path: 'settings/:streamId/',
```

Comment:
This smells a bit, keep the stream's settings under the stream prefix. So basically: `/stream/${id}/settings` + children routes.

Why? Basically namespacing for the future, eg. when/if we'll have projects, things will get confusing or inconsistent...

Reply:
changed it to your second suggestion of `settings/streams/:streamId` !
did not use child route of `stream/:streamId` due to not wanting to have the stream sidebar 
✔ fixed